hayeon/programmers-1/3진법-뒤집기.js

```diff
@@ -0,0 +1,42 @@
+function solution(n) {
+  const ternary = n.toString(3);
+  const reversed = ternary.split("").reverse().join("");
+  const answer = parseInt(reversed, 3);
+
+  return answer;
+}
```

Comment:
```suggestion
function solution(n) {
  return parseInt(n.toString(3).split("").reverse().join(""), 3);
}
```
- 잘푸셨어요!! 이렇게 한 줄로 끝내는것도 가능하니 참고해주세요 👍

Reply:
우와....한 줄도 가능하군요.....변수명 고민해보겠습니닷!!!!! 감사합니다!! 🙂